tools/plugins/com.liferay.ide.sdk.core/src/com/liferay/ide/sdk/core/SDK.java

```diff
@@ -661,6 +695,48 @@ public String getVersion()
         return version;
     }
 
+    private String getPluginFolder( String type )
```

Comment:
Hi Greg, I didn't find any code existing for this.Actually there is function "getPluginTypeSuffix" in NewLiferayPluginProjectOpMethods, but I can't use this one in sdk class.